### PR TITLE
doc(ja): translate `versioning` into Japanese

### PR DESCRIPTION
**Pull request by yossydev**
## Summary

Partial work of https://github.com/biomejs/biome/issues/880.  `internals/versioning`

## Test Plan

- Check if all links works as expecetd
- Check if all translations make sense

### CODE REVIEW

File: website/src/content/docs/ja/internals/versioning.mdx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+description: Biomeのバージョニング方法について
+---
+
+lint ルールやフォーマットレイアウトなどの修正により、スクリプトが通らなくなる可能性があります。これらの変更の性質上、範囲演算子を使うのではなく、正確なバージョンを package.json に保存することを強く推奨します。
+
+この方法によって、スクリプトが予期せず失敗することがなくなります。
+
+## セマンティックバージョニング
+
+Biomeはセマンティックバージョニングに従っている。Biome はツールチェーンという性質上、どのような変更がメジャー、マイナー、パッチとみなされるかが不明確な場合があります。そのため、Biome は以下のバージョンガイドを使用しています：
```
Comment (unvalley) [inline website/src/content/docs/ja/internals/versioning.mdx:12]:
```suggestion
Biomeはセマンティックバージョニングに従っています。Biome はツールチェーンという性質上、どのような変更がメジャー、マイナー、パッチとみなされるかが不明確な場合があります。そのため、以下のバージョンガイドを使用しています：
```

File: website/src/content/docs/ja/internals/versioning.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+* 確立された構文のフォーマットの変更
+
+### マイナーリリース
+* 新しいルールの追加、またはデフォルトでは推奨されない既存のlintルールを安定したグループに昇格
+* 最近導入された言語機能のlintとフォーマットのサポートを追加（これによりlintエラーが増える場合もあります）
+* 推奨ルールの削除
+* 既存ルールの非推奨化
+* フォーマットを変更したり、lintエラーを増やすことなく、新たな設定オプションを追加
+* 新しい推奨 lint ルールの追加、または既存の lint ルールを nursery グループから stable グループの推奨 lint ルールに昇格
+* 「nursery」以外のルールの削除、またはルールを「nursery」グループに降格
```
Comment (unvalley) [inline website/src/content/docs/ja/internals/versioning.mdx:34]:
```suggestion
* 最近導入された言語機能の lint と format のサポート追加（これにより lint エラーが増える場合があります）
* 推奨ルールの削除
* 既存ルールの非推奨化
* format や lint 結果に影響のない新たな設定オプションの追加
* 新しい推奨 lint ルールの追加、または既存の lint ルールを nursery グループから stable グループの推奨 lint ルールに昇格
* nursery 以外のルールの削除、nursery グループへの降格
```

File: website/src/content/docs/ja/internals/versioning.mdx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+### メジャーリリース
+* フォーマットが変わったり、lintエラーが増えるような設定の変更（オプションの追加/削除、デフォルト値の変更）
+* Biomeの公開APIへの変更
+* 一部のスポットライトを必要とする新機能やツールの昇格
```
Comment (unvalley) [inline website/src/content/docs/ja/internals/versioning.mdx:39]:
```suggestion
* format が変わったり、lint エラーが増えるような設定の変更（オプションの追加/削除、デフォルト値の変更）
* Biome の公開 API への変更
* 一部のスポットライトを必要とする新機能やツールの昇格
```

File: website/src/content/docs/ja/internals/versioning.mdx [hunk 4]
```diff
@@ -0,0 +1,44 @@
+---
+title: バージョニング
+description: Biomeのバージョニング方法について
+---
+
+lint ルールやフォーマットレイアウトなどの修正により、スクリプトが通らなくなる可能性があります。これらの変更の性質上、範囲演算子を使うのではなく、正確なバージョンを package.json に保存することを強く推奨します。
```
Comment (unvalley) [inline website/src/content/docs/ja/internals/versioning.mdx:6]:
```suggestion
lint ルールや format レイアウトなどの修正により、スクリプトが通らなくなる可能性があります。これらの変更の性質上、範囲演算子を使うのではなく、正確なバージョンを package.json に保存することを強く推奨します。
```

**Review by TaKO8Ki — COMMENTED** [review]
After resolving suggestions from unvalley, it looks good to me.

File: website/src/content/docs/ja/internals/versioning.mdx (hunk 3 above)
Comment (yossydev) [reply website/src/content/docs/ja/internals/versioning.mdx:39]: I also felt that the Japanese was a bit strange. How about "特に注目を集めるべき新機能やツールの昇格"?

Comment (yossydev) [reply website/src/content/docs/ja/internals/versioning.mdx]: @unvalley 
Thank you!
[fix: update versioning documentation](https://github.com/biomejs/biome/pull/906/commits/7f385a07022e0bd1ad4a91fa86a71ee9e661793c) Fixed!

**Review by unvalley — APPROVED** [review]
Thank you!

**Pull request merged**